src/eigen/lobpcg_hyper_impl.jl

```diff
@@ -150,7 +150,7 @@ end
 
 # Randomize the columns of X if the norm is below tol
 function drop!(X, tol=2eps(real(eltype(X))))
-    dropped = []
+    dropped = Vector{Int}()
```

Comment:
Just use `Int[]`. I've started to prefer that over the long version.

Reply:
Cool, did not know